src/algorithms/calorimetry/CalorimeterClusterMerger.cc

```diff
@@ -39,31 +39,32 @@ void CalorimeterClusterMerger::AlgorithmProcess() {
     auto& assoc2 = m_outputAssociations;
 
     if (!split.size()) {
-      if (false) {
-        LOG_INFO(default_cout_logger) << "Nothing to do for this event, returning..." << LOG_END;
+      if (m_logger->level() <=spdlog::level::debug) {
```

Comment:
This comparison is not needed if you just do `m_logger->debug(...)`. But in terms of functioning, this is good now

Reply:
desire to leave the code as close to the original for now